deploy/datadog-agent/etc/datadog-agent/conf.d/json_check.md

```diff
@@ -30,6 +33,8 @@ init_config:
 instances:
     - url: https://api.coinmarketcap.com/v2/ticker
       metrics: 
-        - usd.price: data."1".quotes.USD.price
+      # Numeric key names should be in double quotes, otherwise they will be counted as an array index
+        - usd.price: data."1".quotes.USD.price OR [data]["1"][quotes][USD][price]
```

Comment:
this OR is illegal. just add another example for EUR or something and use the second notation there